src/System.Private.ServiceModel/tests/Scenarios/Binding/Http/NetHttpsBindingTests.cs

```diff
@@ -5,15 +5,13 @@
 
 using System;
 using System.ServiceModel;
-using System.ServiceModel.Channels;
-using System.Text;
-using System.Threading;
-using System.Threading.Tasks;
+
+using Infrastructure.Common;
 using Xunit;
 
-public static class Binding_Http_NetHttpsBindingTests
+public class Binding_Http_NetHttpsBindingTests: ConditionalWcfTest
 {
-    [Fact]
+    [ConditionalFact(nameof(Root_Certificate_Installed))]
```

Comment:
ConditionalFact is not recognized by ToF, which means adding this attribute will cause this test to be skipped silently.

An alternative is to keep the [Fact] attribute and add a direct call like Assert.True(Root_Certificate_Installed(), "Root cert is not installed");

We can discuss how to do this more generally separately.  I'm just observing this change will effectively disable the tests in ToF for both CoreCLR and NET Native.